src/exos-scripts.ts

```diff
@@ -1,9 +1,7 @@
 import chalk from 'chalk';
-import path from 'path';
-import spawn from 'cross-spawn';
 import { ExosScripts } from './common/types';
 
-export default function exosScripts(scriptName: string, args: string[]): void {
+export default function exosScripts(scriptName: string): void {
```

Comment:
Why are you removing the args?

Reply:
Because currently there is 2 process spawns. It should be improved but in different PR. Now I'm doing this minimum to make it possible for tests to work without prebuild.

Reply:
Usually babel, transpile, etc. is handled by Jest setup. However once we have functional unit tests, we probably will want to build for that script such as `npm run test:functional` or `npm run test:browser`. 

Would a `npm run test:unit` be a good option?